webapp/src/pages/Map.tsx

```diff
@@ -731,6 +845,12 @@ function MapPage() {
       type: 'symbol',
       tag: 'intersection',
     },
+    {
```

Comment:
This can be a backlog task, but it would be nice to have a legend with colors on the map

Reply:
I went ahead and added this now